.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "uwrt_mars_rover_hw_bridge"]
+	path = uwrt-mars-rover-hw-bridge
+	url = https://github.com/uwrobotics/uwrt_mars_rover_hw_bridge.git
```

Comment:
use a relative path here so if someone uses ssh to clone repo, they use ssh to clone submodule

Reply:
i couldn find anything on how to do this

Reply:
https://www.gniibe.org/memo/software/git/using-submodule.html

Reply:
just google "git submodule relative path"